litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/index.tsx

```diff
@@ -215,6 +216,14 @@ const WorkflowSettings = forwardRef((_, ref) => {
                 onChange={nameChangeHandle}
                 value={name}
               />
+              <InputField
+                InputProps={{
+                  readOnly: true,
+                }}
+                className={classes.nsInput}
+                label={t('createWorkflow.chooseWorkflow.label.namespace')}
+                value={workflowData.namespace}
+              />
             </div>
             <div aria-details="spacer" style={{ margin: '3rem 0' }} />
```

Comment:
same here 🙂

Reply:
Here its not okay as spacing should be used from theme for all margin and paddings.